packages/backend/src/graphql/queries/test-connection.ts

```diff
@@ -3,25 +3,56 @@ import App from '@/models/app'
 import Context from '@/types/express/context'
 
 type Params = {
-  id: string
-  data: object
+  connectionId: string
```

Comment:
We set CF's browser cache TTL to 4 hours, so in the worst case, we can get stale clients sending the old query (with `id` and `data` params) for 4 hours.

I don't think it's a big an issue since line 27 should make these clients error out, but it's something we should note at release in case we get any support queries about errors when testing connections.

And we need to remember to flush CF cache after releasing too..

Reply:
CF doesnt cache index.html, and when a new react build is generated, it has a different bundle name based on some checksum (e.g. bundle.abc123.js or sth like this). When index.html imports this via <script/> it will not import the old bundle. 

But of course, this means that users who have pre-loaded plumber.gov.sg before the release will be facing 400 errors and they just need to refresh.

Reply:
Wah swee I didn't know that. Ya I was thinking about the pre-loaded users (i.e. access plumber right before we release)